### PR TITLE
Add Phone button

### PR DESCRIPTION
**Pull request by nicmcr**
# Proposed Changes
Added new button for a simple phone number. 

image

## Checklist

- [✅ ] Tested locally
- [✅] Ran `yarn ci` to test my code
- [✅] Did not add any unnecessary changes
- [✅] All my changes appear after other changes in each file
- [✅] Included a screenshot (if adding a new button)
- [ ] 🚀

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,5 @@ build
 .env.test.local
 .env.production.local
 .env
-cache
\ No newline at end of file
+cache
+.gitpod.yml
```
Comment (timothystewart6) [inline .gitignore:15]: out of curiosity, why the change here?

**Comment by timothystewart6** [comment]
Hey, thank you for this.  Since this is not a brand, have you considered a custom button?  This is exactly the use case this tries to solve - a button that has an icon that says whatever you like, that does whatever you like.

**Comment by nicmcr** [comment]
Hi Tim! This is my first PR so it's a little bit 

image

Didnt realise there was a custom button option! Agree that that makes more sense. Is the custom button documented anywhere? It's not in `docker-compose.yml`. 

Thanks

**Comment by timothystewart6** [comment]
Haha!  No worries!  It's the in the readme but not documented as well as it could be!   This is awesome and you did it exactly right, but I think you'll find more flexibility with using a custom button!

**Comment by nicmcr** [comment]
Thanks I will check it out. Love your work. 

Cheers

**Comment by timothystewart6** [comment]
Thank you!  Let me know how it works out!  post a screenshot after you try it!
